src/features/completions.zig

```diff
@@ -1345,7 +1354,15 @@ pub fn collectContainerFields(
     for (container_decl.ast.members) |member| {
         const field = handle.tree.fullContainerField(member) orelse continue;
         const name = handle.tree.tokenSlice(field.ast.main_token);
-        try completions.append(arena, .{
+        if (likely == .enum_assignment and !field.ast.tuple_like and container.isTaggedUnion()) {
+            try completions.append(arena, .{
+                .label = name,
+                .kind = if (field.ast.tuple_like) .EnumMember else .Field,
+                .detail = Analyser.getContainerFieldSignature(handle.tree, field),
+                .insertText = if (snippets_ok) try std.fmt.allocPrint(arena, "{{ .{s} = $1 }}$0", .{name}) else try std.fmt.allocPrint(arena, "{{ .{s} = ", .{name}),
```

Comment:
I am a bit unsure about this but should this feature be enabled even snippets are disabled? I would have expected to retain the old behavior when snippets are disabled.

Reply:
For me snippets is the `$1 $2 $0` functionality, otherwise it's a valid insertText.